Automattic/jetpack
# [Jetpack AI] Unleash

This PR Releases the Jetpack AI for all WPCOM Users


## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
If you are an Automattician, nothing should change

* Sync this branch to a wpcom site,
* search for blocks tagged with "Magic"
* You should see them without the "Beta" designation
* You should be able to use them
